functions/src/etl/StripeWebhook.ts

```diff
@@ -153,7 +154,7 @@ export class StripeWebhook {
 		const balanceTransaction = charge.balance_transaction as Stripe.BalanceTransaction;
 		return {
 			source: ContributionSourceKey.STRIPE,
-			created: new Date(charge.created * 1000),
+			created: new Timestamp(charge.created, 0),
```

Comment:
@andrashee: We need to make sure this works when we merge this. I would hope so.